dist/etc/conf/arduino-due.env

```diff
@@ -1,72 +1,77 @@
-RF_DUT_RST_GPIO=
-RF_DUT_RST_PORT=
-RF_DUT_RST_PIN=
-
-RF_DUT_CTS_GPIO=
-RF_DUT_CTS_PORT=
-RF_DUT_CTS_PIN=
-
-RF_DUT_RTS_GPIO=
-RF_DUT_RTS_PORT=
-RF_DUT_RTS_PIN=
-
-RF_DUT_RX_GPIO="PA11/TX1"
-RF_DUT_RX_PORT=0
-RF_DUT_RX_PIN=11
-
-RF_DUT_TX_GPIO="PA10/RX1"
-RF_DUT_TX_PORT=0
-RF_DUT_TX_PIN=10
-
-RF_DUT_NSS_GPIO="PD8/12"
-RF_DUT_NSS_PORT=3
-RF_DUT_NSS_PIN=8
-
-RF_DUT_SCK_GPIO="PA27/SPI-3"
-RF_DUT_SCK_PORT=0
-RF_DUT_SCK_PIN=27
-
-RF_DUT_MISO_GPIO="PA25/SPI-1"
-RF_DUT_MISO_PORT=0
-RF_DUT_MISO_PIN=25
-
-RF_DUT_MOSI_GPIO="PA26/SPI-4"
-RF_DUT_MOSI_PORT=0
-RF_DUT_MOSI_PIN=26
-
-RF_DUT_SCL_GPIO=
-RF_DUT_SCL_PORT=
-RF_DUT_SCL_PIN=
-
-RF_DUT_SDA_GPIO=
-RF_DUT_SDA_PORT=
-RF_DUT_SDA_PIN=
-
-RF_DUT_PWM_GPIO=
-RF_DUT_PWM_PORT=
-RF_DUT_PWM_PIN=
-
-RF_DUT_DAC_GPIO=
-RF_DUT_DAC_PORT=
-RF_DUT_DAC_PIN=
-
-RF_DUT_ADC_GPIO=
-RF_DUT_ADC_PORT=
-RF_DUT_ADC_PIN=
-
-RF_DUT_IC_GPIO=
-RF_DUT_IC_PORT=
-RF_DUT_IC_PIN=
-
-RF_DEBUG0_GPIO="P18/SCL1"
-RF_DEBUG0_PORT=1
-RF_DEBUG0_PIN=8
-
-RF_DEBUG1_GPIO="P17/SDA1"
-RF_DEBUG1_PORT=1
-RF_DEBUG1_PIN=7
-
-RF_DEBUG2_GPIO="P27/13"
-RF_DEBUG2_PORT=1
-RF_DEBUG2_PIN=7
-
+HIL_DUT_RST_GPIO?=
+HIL_DUT_RST_PORT?=
+HIL_DUT_RST_PIN=
```

Comment:
why no `?=` for the pins? IIRC all should be defined that way to allow for overrides from command line

Reply:
up, missed it!